source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -285,6 +297,38 @@ class StreamHandleWrapper : public Filters::Common::Lua::BaseLuaObject<StreamHan
   State state_{State::Running};
   std::function<void()> yield_callback_;
   Http::AsyncClient::Request* http_request_{};
+  FireAndForgetWriter* fireAndForgetWriter_;
+};
+
+/**
+ * Implementation that takes incoming requests and implements
+ * "fire and forget" behavior using an async client.
+ */
+class FireAndForgetWriter : public Filters::Common::Lua::BaseLuaObject<FireAndForgetWriter>,
```

Comment:
Since this writer currently is an HTTP async client "writer". Can we name it as `FireAndForgetHttpWriter` or `HttpAsyncClientWriter` or similar?